applications/FluidDynamicsApplication/custom_processes/weighted_divergence_calculation_process.h

```diff
@@ -0,0 +1,338 @@
+//    |  /           |
```

Comment:
Why is this header only? If there is no technical necessity, please separate in .h and .cpp files.

Reply:
I splitted into .h and .cpp. If you can go quickly over it just to check everything is in the right place, thanks!